guide/src/main/assets/pages/ext/i18n.md

```diff
@@ -0,0 +1,116 @@
+# Udash i18n
```

Comment:
I don't know why but after I've deleted that file, none of the `MarkdownPage`s rendered properly :/

Reply:
Cannot reproduce, maybe you had backend off?